go/vt/binlog/binlogplayer/mock_dbclient.go

```diff
@@ -224,3 +224,16 @@ func (dc *MockDBClient) ExecuteFetch(query string, maxrows int) (qr *sqltypes.Re
 	}
 	return result.result, result.err
 }
+
+func (dc *MockDBClient) ExecuteFetchMulti(query string, maxrows int) ([]*sqltypes.Result, error) {
+	queries := strings.Split(query, ";")
```

Comment:
I guess this is only used in testing, since this wouldn't be safe to do otherwise? Should we use the parser here to make sure we're safe even if it's only testing code?

Reply:
Good point. No reason not to use the parser. I'll update these spots to use:
```
sqlparser.SplitStatementToPieces(query)
```